README.md

```diff
@@ -1,70 +1,57 @@
-# mbed-os-example-psa
+# PSA Mbed OS example
 
-This is an example application showing how to query the PSA version. It also blinks an LED to indicate it is working.
+This is an example application showing how to query the PSA version.
 
-## Prerequisite
+You can build this project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
 
-* [Mbed CLI](https://github.com/ARMmbed/mbed-cli) - to build the example program. For installation and usage instructions, see the [documentation](https://os.mbed.com/docs/mbed-os/latest/build-tools/mbed-cli.html).
+1. Install Mbed CLI.
+1. From the command-line, import the example: `mbed import mbed-os-example-psa`
+1. Change the current directory to where the project was imported.
 
-## Building the Mbed OS application
+## Application functionality
 
-#### Import the example
+The `main()` function queries the PSA version and toggles the state of a digital output connected to an LED on the board.
 
-1. From the command line, clone the repository containing the example:
-    ```
-    $ git clone https://github.com/ARMmbed/mbed-os-example-psa
-    ```
+## Building and running
 
-1. Change the current directory to where the project was imported.
+1. Connect a USB cable between the USB port on the target and the host computer.
+1. Run the following command to build the example project, program the microcontroller flash memory, and open a serial terminal:
 
-1. Update `mbed-os` sources by running `mbed deploy`.
+   ```
+   $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm --baud 115200
```

Comment:
We could use give target and compiler in the example so that user can just copy-paste the snippet, Please remove `$` in that case